multichain-testing/test/auto-stake-it.test.ts

```diff
@@ -96,9 +100,9 @@ const autoStakeItScenario = test.macro({
     const fundAndTransfer = makeFundAndTransfer(t);
 
     // 2. Find 'stakingDenom' denom on agoric
-    const remoteChainInfo = (chainInfo as Record<string, CosmosChainInfo>)[
-      chainName
-    ];
+    const remoteChainInfo = (
+      starshipChainInfo as Record<string, CosmosChainInfo>
+    )[chainName];
```

Comment:
the cast is no longer necessary https://github.com/Agoric/agoric-sdk/blob/ef82fbf7466ea1b57c321f735d73fcee2c66fd11/multichain-testing/starship-chain-info.d.ts#L1-L3

```suggestion
    const remoteChainInfo = starshipChainInfo[chainName];
```